src/main/java/org/mamute/list/TabsHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.mamute.list;
+
+import br.com.caelum.vraptor.http.route.Router;
+import net.vidageek.mirror.dsl.Mirror;
+import org.apache.commons.lang.WordUtils;
+import org.mamute.controllers.ListController;
+
+import javax.inject.Inject;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class TabsHelper {
```

Comment:
Well the stuff implemented here is not a model business logic per-se, it is rather a mapping of the information model shouldn't know about. Think of a database model doing its own database connection. You would usually abstract that out and inject the values into the model once you fetch them. It doesn't mean that you introduced a anemic model by doing this. Again, your repo, your rules, so let me know if you insist on this :)